apps/frontend-web/app/route-guard.tsx

```diff
@@ -0,0 +1,16 @@
+'use client';
+import { useAuth } from '@test-assessment/ui-auth-protect';
+import { useRouter } from 'next/navigation';
+import { useEffect } from 'react';
+
+export default function RouteGuard({ children }) {
+  const auth = useAuth();
+  const router = useRouter();
+
+  useEffect(() => {
+    if (auth.loading && !auth.user) {
+      router.push('/login');
+    }
+  }, [auth]);
```

Comment:
only check auth.loading or auth.user. probably auth will have many properties can be changed

Reply:
yep, removed auth.loading